### DISCUSSION TITLE
[DOC] How to run bunkerweb behind existing reverse proxy (NPM) ?

### DISCUSSION THREAD

**Discussion by Mag1cByt3s** [bunkerity/bunkerweb#608] [Help] [answered]
**Description**

I am trying to figure out how i can deploy bunkerweb behind or in front of an existing reverse proxy.
In my case I am running Nginx Proxy Manager (NPM). At the moment i am forwarding Port 80 and 443 directly to NPM.

I looked through the documentation but could not get this setup working. Can someone help with that please?

**Comment by usmankhawar22** [comment]
Looking for the same, behing ingress-nginx

**Comment by tedimacko** [comment]
A little guide to the setup would be nice.

**Comment by alfi4000** [comment]
Any update,

**Comment by TheophileDiot** [comment, marked as answer]
Hi guys, I'm digging this up to share that link: https://docs.bunkerweb.io/latest/advanced/#behind-load-balancer-or-reverse-proxy
